libRALFit/src/ral_nlls_workspaces.f90

```diff
@@ -571,14 +584,14 @@ type, public :: get_svd_J_work ! workspace for subroutine get_svd_J
   type, public :: NLLS_workspace ! all workspaces called from the top level
      logical :: allocated = .false.
      integer :: first_call = 1
-     integer :: iter = 0 
+     integer :: iter = 0
      real(wp) :: normF0, normJF0, normF, normJF
      real(wp) :: normJFold, normJF_Newton
      real(wp) :: Delta
-     real(wp) :: normd
+     real(wp) :: normd = 1.0e10_wp
```

Comment:
Why is `normd` set to this value here?

Reply:
`normd` is now undefined and `test_convergence` for iteration 1 only tests for objective and gradient conditions.